apps/web/components/eventtype/EventTeamTab.tsx

```diff
@@ -394,10 +398,29 @@ export const EventTeamTab = ({
     );
   });
   const isManagedEventType = eventType.schedulingType === SchedulingType.MANAGED;
+  const [isOpenAssignmentWarnDialog, setIsOpenAssignmentWarnDialog] = useState<boolean>(false);
+  const [pendingRoute, setpendingRoute] = useState("");
+  const isConfirm = useRef(false);
   const { getValues, setValue } = useFormContext<FormValues>();
   const [assignAllTeamMembers, setAssignAllTeamMembers] = useState<boolean>(
     getValues("assignAllTeamMembers") ?? false
   );
+  const router = useRouter();
+
+  useEffect(() => {
+    const handleRouteChange = (url: string) => {
+      if (getValues("hosts").length === 0 && getValues("children").length === 0 && !isConfirm.current) {
+        setIsOpenAssignmentWarnDialog(true);
+        setpendingRoute(url);
+        router.events.emit("routeChangeError", new Error(`Aborted route change to ${url}`));
```

Comment:
Shall it be a more descriprive error - aborter rotue change to ${url} because noone was assigned to team event?